internal/hud/webview/convert.go

```diff
@@ -193,11 +193,6 @@ func ToUIResourceList(state store.EngineState) ([]*v1alpha1.UIResource, error) {
 		ret = append(ret, TiltfileResourceProtoView(name, ms, state.LogStore))
 	}
 	for i, mt := range state.Targets() {
-		// Skip manifests that don't come from the tiltfile.
-		if mt.Manifest.Source != model.ManifestSourceTiltfile {
```

Comment:
Is there anything special on the UI side needed for these to work properly? (It seems like it should just work to me - just checking)